### ISSUE TITLE
Cannot select fruit to transport for truck MAN TGS 18.500 4x4

### ISSUE THREAD

**Issue by Dennizzzzz** [Courseplay/courseplay#5706]
I cannot select the type of fruit to transport for truck MAN TGS 18.500 4x4. 
The truck is a mod, so i'm not sure it is a bug or an incompatible mod.

![2020-07-04 18_26_45-Farming Simulator 19](https://user-images.githubusercontent.com/10248136/86516907-dcc1a600-be24-11ea-998d-ca0dc54fba21.jpg)
![2020-07-04 18_27_19-Farming Simulator 19](https://user-images.githubusercontent.com/10248136/86516910-e3e8b400-be24-11ea-9387-d026dbd868ac.jpg)

[log.txt](https://github.com/Courseplay/courseplay/files/4873212/log.txt)

Courseplay for FS19 v6.02.00037
Map: RusticAcres_V3

**Comment by Tensuko** [comment]
Can you try out other Mods that are similar ?

**Comment by Dennizzzzz** [comment]
Indeed other same sort of trucks have the same issue. Did try to find which version did work, tried backwards thru the versions, and every version from v6.02.00031 to v6.02.00037 has the same problem. Did NOT go further down the version...

**Comment by rohne83** [comment]
I noticed that this truck has problems especially with chaff. It will have its fillUnit only be filled to about 88%, and then everything stops because the chopper still tries to load into the truck, but for unknown reason the truck is not able to take more load (although still about 10% of capacity left). This only happens when chopping though. No problems with loading grain or at a silo. Could this have a related reason as the problem here with fruit selection?

**Comment by Tensuko** [comment]
@rohne83 did you test with .00071 ?
I have no time to try all the last Issues with newest Versions. Would be nice if you have some feedback for this one.

**Comment by rohne83** [comment]
Hi, just tested with v71. Cannot reproduce the original issue here. Fruit types for refilling can be selected.
![fsScreen_2020_10_23_11_49_33](https://user-images.githubusercontent.com/71030138/97082389-94173e00-1609-11eb-8f49-ee23eb470dec.png)
Cannot speak for the opener of this issue, but in my eyes it is solved.
And also found the root cause for the chopping issue which I added two days ago. It doesn't have anything to do with Courseplay of the fruitType selection. Just for those who got frustrated facing the same issue: the reason is the endZ value for autoAimTargetNode in the MAN's xml. The modder had set it to -2.5 which results in the pipe aiming behind the trailer when the fillLevel reaches higher values. Setting endZ to -0.5 solved that issue.

**Comment by Tensuko** [comment]
Awesome! Thank you for Testing this.
Your information about the autoAimTargetNode is also very helpful.
Wich MAN is this exaclty ? ModHub one ?

**Comment by rohne83** [comment]
Yes. "Man TGS AgroTruck and Kröger HKD Pack" by Kasztan. But I noticed there are several mods even on ModHub for that truck with its various different versions. So not sure whether issue opener is using the exact same mod as me.

**Comment by Dennizzzzz** [comment]
Hi, just looked for the mod I use, and yes it:s the one described in the previous comment. So it is fixed.
